Resources/Audio/BoomBox/sources.txt

```diff
@@ -0,0 +1,3 @@
+the song is taken from the YouTube resource.
+AssHole: https://www.youtube.com/watch?v=y4ADXLk3weU
+TheWizard: https://www.youtube.com/watch?v=jJDAV9vSmYc
```

Comment:
1. We already have space asshole
2. Use attributions.yml rather than doing this.
3. I don't think "TheWizard" is also licensed correctly. Please respect the licensing.